app/views/mentors/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1>Mentors</h1>
+
+<!-- Mentor Table -->
+<table class='highlight responsive-table'>
+  <thead>
```

Comment:
what do you think about surrounding the table in a container tag. this will keep the views standard and not butted up against the side and top and bottom
```html
<div class='container'>
    <table>
        ....
    </table>
</div>
```